libraries/messages/src/lib.rs

```diff
@@ -24,7 +24,10 @@ pub mod sensor;
 pub mod sensor_status;
 pub mod state;
 
-pub const MAX_SIZE: usize = 64;
+pub const MAX_HEALTH_SIZE: usize = 47;
+pub const MAX_SENSOR_SIZE: usize = 53;
+pub const MAX_STATE_SIZE: usize = 13;
+pub const MAX_LOGNCOMMAND_SIZE: usize = 15;
```

Comment:
I know they are the same size, but they might not always be the same size, can you split these.